pupil_src/capture/run_capture.bat

```diff
@@ -1,2 +1,2 @@
-PATH %PATH%..\..\pupil_external
+SET PATH=%PATH%;%~dp0..\..\pupil_external
```

Comment:
@PolynomialDivision - AFAIK using `SET` will add this path to the environment variables. 

This batch script without `SET` is intended to be used to add the `pupil_external` `.dll`s to the path only for the current session and not permanently. 

Regarding the addition of `%~dp0` - this will expand the path to the drive letter and the folder from which run_capture.bat resides. Example: `C:\work\pupil\pupil_src\capture` - so this seems like it makes sense to me. However, adding the extra `;` after `%PATH%` seems redundant (at least in my test it adds a double `;` in the path where only one is needed.

Can you please try:

`PATH %PATH%%~dp0..\..\pupil_external`